src/tbb/co_context.h

```diff
@@ -57,7 +68,19 @@ namespace tbb {
 namespace detail {
 namespace r1 {
 
+#if __TBB_RESUMABLE_TASKS_USE_THREADS
+    struct coroutine_type {
 #if _WIN32 || _WIN64
+        HANDLE my_thread;
+#else
+        pthread_t my_thread;
+#endif
```

Comment:
Could "handle_type" type be used instead?

Reply:
@vlserov check it again, please.